service/src/main/java/bio/terra/catalog/datarepo/DatarepoService.java

```diff
@@ -19,6 +22,15 @@
 @Component
 public class DatarepoService {
   private static final Logger logger = LoggerFactory.getLogger(DatarepoService.class);
+  public static final String ADMIN_ROLE_NAME = "admin";
+  public static final String CUSTODIAN_ROLE_NAME = "custodian";
+  public static final String READER_ROLE_NAME = "reader";
+  public static final String DISCOVERER_ROLE_NAME = "discoverer";
+
+  private static final List<String> OWNER_ROLES = List.of(ADMIN_ROLE_NAME, CUSTODIAN_ROLE_NAME);
```

Comment:
Snapshots also have the steward role. The snapshot creator automatically becomes a steward and they can add other users as stewards.